main.go

```diff
@@ -17,10 +17,14 @@ import (
 	"log"
 	"os"
 
+	"go.uber.org/automaxprocs/maxprocs"
+
 	"github.com/gohugoio/hugo/commands"
 )
 
 func main() {
+	// default GOMAXPROCS to be CPU limit aware, still respecting GOMAXPROCS env
+	maxprocs.Set()
```

Comment:
Could you 

1. Move this into commands.Execute so it actually gets executed during the integration tess.
2. Make the comment into a proper sentence starting with a capital D and ending with a period.
3. Adjust the commit message title in line with the guidelines in this repo (start out with a capital A)

Reply:
Also, it would be good if you could rebase it against the recently pushed master.

Reply:
All three should be done.